<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>attendance_screen</class>
 <widget class="QMainWindow" name="attendance_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>440</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background: url(:/ui/main_bg.jpeg)  no-repeat center center fixed;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>5</y>
      <width>450</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/ui/attendance.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Enter_details">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);
font: 70 24pt &quot;Tahoma&quot;;</string>
    </property>
    <property name="text">
     <string>Enter The Subject Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sub_name_input">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="subject_name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 21pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Subject Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Fill_attendance">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(170, 255, 255);
	font: 75 18pt &quot;Times New Roman&quot;;
	border-radius: 12px;
}
QPushButton:hover
{font: 75 20pt &quot;Times New Roman&quot;;
	background-color: rgb(75, 225, 225);
}
QPushButton:pressed 
{
	background-color: rgb(52, 143, 255);
}</string>
    </property>
    <property name="text">
     <string>Fill Attendance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chech_sheets">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(170, 255, 255);
	font: 75 18pt &quot;Times New Roman&quot;;
	border-radius: 12px;
}
QPushButton:hover
{font: 75 20pt &quot;Times New Roman&quot;;
	background-color: rgb(75, 225, 225);
}
QPushButton:pressed 
{
	background-color: rgb(52, 143, 255);
}</string>
    </property>
    <property name="text">
     <string>Check Sheets</string>
    </property>
   </widget>
   <widget class="QLabel" name="Output">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>700</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 70 19pt &quot;Tahoma&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="registerscreen.qrc"/>
 </resources>
 <connections/>
</ui>
